README.md

```diff
@@ -1,2 +1,20 @@
-# hw01
-Repository for first CFSS homework assignment
+# Cheng Yee Lim
+
+![](images/profile_pic.jpg)
```

Comment:
Good use of adding an image into the repo itself rather than linking to an external image